src/celeritas/inp/PhysicsProcess.hh

```diff
@@ -90,6 +90,82 @@ using MuPairProductionProcess = PairProductionProcess;
 
 //!@}
 
+//---------------------------------------------------------------------------//
+/*!
+ * Muon-catalyzed fusion mean cycle rate data.
+ *
+ * Mean cycle rates are as a function of temperature, with each grid assigned
+ * to a muonic molecule and its spin (e.g. \f$ (dt)_\mu, F = 0 \f$).
+ *
+ * The inverse of the rate provides the time it takes for a muon-catalyzed
+ * fusion to happen, from atom formation to fusion. This number is used to
+ * update the track time in the stepping loop and allows the competition
+ * between fusion and muon decay.
+ *
+ * Since the cycle rate is per-material, these grids are host-only, with only
+ * the final cycle rate (a \c real_type ) for each molecule + spin being needed
+ * in the stepping loop.
+ */
+struct CycleRateData
```

Comment:
These should have a Muon prefix I think, and avoid calling it "Data" since it's implicitly input parameters and it's not the other device "Data" that we use. I think `MuonicFusionRate` ought to do it.